acceptance/bundle/deploy/registered_models/basic/output.txt

```diff
@@ -0,0 +1,132 @@
+
+Plan: 0 to add, 1 to change, 0 to delete, 0 unchanged
+
+>>> [CLI] bundle deploy
+Uploading bundle files to /Workspace/Users/[USERNAME]/.bundle/deploy-registered-models-basic-[UNIQUE_NAME]/default/files...
+Deploying resources...
+Updating deployment state...
+Deployment complete!
+
+>>> [CLI] registered-models get main.default.my-registered-model-[UNIQUE_NAME]
+{
+  "name": "my-registered-model-[UNIQUE_NAME]",
+  "comment": "updated comment",
+  "catalog_name": "main",
+  "schema_name": "default"
+}
+
+=== update the name, this should recreate
+>>> [CLI] bundle plan
+recreate registered_models.my_registered_model
+
+Plan: 1 to add, 0 to change, 1 to delete, 0 unchanged
+
+>>> [CLI] bundle deploy
+Uploading bundle files to /Workspace/Users/[USERNAME]/.bundle/deploy-registered-models-basic-[UNIQUE_NAME]/default/files...
+Deploying resources...
+Updating deployment state...
+Deployment complete!
+
+>>> [CLI] registered-models get main.default.my-registered-model-updated-[UNIQUE_NAME]
+{
+  "name": "my-registered-model-updated-[UNIQUE_NAME]",
+  "comment": "updated comment",
+  "catalog_name": "main",
+  "schema_name": "default"
+}
+
+=== update the catalog name, this should recreate
+>>> [CLI] bundle plan
+recreate registered_models.my_registered_model
+
+Plan: 1 to add, 0 to change, 1 to delete, 0 unchanged
+
+>>> [CLI] bundle deploy
+Uploading bundle files to /Workspace/Users/[USERNAME]/.bundle/deploy-registered-models-basic-[UNIQUE_NAME]/default/files...
+Deploying resources...
+Updating deployment state...
+Deployment complete!
+
+>>> [CLI] registered-models get mycatalog-[UNIQUE_NAME].default.my-registered-model-updated-[UNIQUE_NAME]
+{
+  "name": "my-registered-model-updated-[UNIQUE_NAME]",
+  "comment": "updated comment",
+  "catalog_name": "mycatalog-[UNIQUE_NAME]",
+  "schema_name": "default"
+}
+
+=== update the schema name, this should recreate
+>>> [CLI] bundle plan
+recreate registered_models.my_registered_model
+
+Plan: 1 to add, 0 to change, 1 to delete, 0 unchanged
+
+>>> [CLI] bundle deploy
+Uploading bundle files to /Workspace/Users/[USERNAME]/.bundle/deploy-registered-models-basic-[UNIQUE_NAME]/default/files...
+Deploying resources...
+Updating deployment state...
+Deployment complete!
+
+>>> [CLI] registered-models get mycatalog-[UNIQUE_NAME].myschema-[UNIQUE_NAME].my-registered-model-updated-[UNIQUE_NAME]
+{
+  "name": "my-registered-model-updated-[UNIQUE_NAME]",
+  "comment": "updated comment",
+  "catalog_name": "mycatalog-[UNIQUE_NAME]",
+  "schema_name": "myschema-[UNIQUE_NAME]"
+}
+
+>>> [CLI] bundle destroy --auto-approve
+The following resources will be deleted:
+  delete registered_model my_registered_model
+
+All files and directories at the following location will be deleted: /Workspace/Users/[USERNAME]/.bundle/deploy-registered-models-basic-[UNIQUE_NAME]/default
+
+Deleting files...
+Destroy complete!
+
+>>> [CLI] schemas delete mycatalog-[UNIQUE_NAME].myschema-[UNIQUE_NAME] --force
```

Comment:
As discussed offline, would be great to use acceptance/bundle/resources/<resource_name>/<subtest or action> schema.

bundle/deploy is really focussed on deploy command but we're also interested in plan/json plan/destroy/summary.

another aspect is to use smaller tests which is good for perf and debugging:

See how this test is organized:

```
 ~/work/cli/acceptance/bundle/resources % tree -d volumes
volumes
├── change-comment
├── change-name
├── change-schema-name
├── remote-change-name
├── remote-delete
└── set-storage-location
```

Reply:
I'll address this in a followup PR.